helm/values.yaml

```diff
@@ -78,4 +78,6 @@ microservice-chart:
     SERVICEBUS_ONBOARDING_CONNECTION_STRING: idpay-onboarding-request-processor-sas-key
     INITIATIVE_BLOB_STORAGE_CONNECTION_STRING: initiative-storage-blob-connection-string
     RANKING_P7M_CERT: ranking-p7m-cert
-    RANKING_P7M_KEY_PEM: ranking-p7m-key
\ No newline at end of file
+    RANKING_P7M_KEY_PEM: ranking-p7m-key
+    PDV_DECRYPT_API_KEY: pdv-api-key
+    PDV_DECRYPT_BASE_URL: pdv_decrypt_base_url
```

Comment:
"pdv_decrypt_base_url" doesn't exists in the keyvault, maybe it belong to a configMap?